pkgs/native_assets_cli/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.3.2
+
+- Fixed an issue where `Depenendencies.dependencies` could not be
+  modified when expected to.
+  **Note:** Avoid modifying `Dependencies.dependencies` directly,
```

Comment:
I think in a future, it Dependencies itself should maybe implement iterable. This whole API needs some iteration:

* https://github.com/dart-lang/native/issues/25

For now, maybe just leave the note out, because we don't know what the API should look like.

Maybe the API on will have `addDependency` and no getter at all.